feature/storage/src/main/java/com/mashup/dorabangs/feature/storage/storagedetail/StorageDetailList.kt

```diff
@@ -59,7 +64,18 @@ fun StorageDetailList(
                 onClickTabItem = onClickTabItem,
                 onClickActionIcon = onClickActionIcon,
             )
-            StorageDetailEmpty(modifier = modifier)
+            if (isLoading.not()) {
+                StorageDetailEmpty(modifier = modifier)
+            } else {
+                Box(modifier = Modifier.fillMaxSize()) {
+                    LottieLoader(
+                        lottieRes = coreR.raw.spinner,
+                        modifier = Modifier
+                            .size(54.dp)
+                            .align(Alignment.Center),
+                    )
+                }
+            }
```

Comment:
ㅋㅋㅋㅋ이것보단 한번 꼬지 말고 아래가 더 읽기 쉬울듯,,
```suggestion
            if (isLoading) {
                Box(modifier = Modifier.fillMaxSize()) {
                    LottieLoader(
                        lottieRes = coreR.raw.spinner,
                        modifier = Modifier
                            .size(54.dp)
                            .align(Alignment.Center),
                    )
                }
            } else {
                StorageDetailEmpty(modifier = modifier)
            }
```

Reply:
넵 수정완~